docs/imagen.md

```diff
@@ -0,0 +1,76 @@
+ - **Seguridad**: Se procurará elegir una imagen con la menor cantidad de vulnerabilidades posibles.
+
+Se descartarán las imágenes que no cuenten con una instalación de Go por defecto, ya que implican un trabajo extra que no considero necesario teniendo en cuenta la gran
+cantidad de opciones.
+
+## [Bitnami](https://hub.docker.com/r/bitnami/golang)
+
+Esta imagen, de la organización *Bitnami*, promete ser segura y pequeña, ya que se basa en una distribución llamada Minideb. Veamos como se comporta en relación a nuestros criterios:
+
+| Criterio              | Ajuste en herramienta | 
+|------------------------|-----------------------|
+| *Popularidad* | La imagen tiene más de medio millón de pulls en Docker Hub. |
+| *Mantenimiento* | La última imagen es de hace 15h (visitado el 7 de febrero de 2024). |
+| *Seguridad* | En DockerHub no aparecen publicadas las vulnerabilidades de la imagen, pero en el [README](https://github.com/bitnami/containers?tab=readme-ov-file#vulnerability-scan-in-bitnami-container-images) del repositorio de sus contenedores describen que utilizan dos herramientas para escanear sus imágenes antes de publicarlas. |
+
+## [Golang Oficial](https://hub.docker.com/_/golang)
+
+El lenguaje de programación tiene varias imágenes oficiales, de las cuales discutiremos la basada en Debian-bullseye y Alpine.
+
+En general, podemos decir que estas opciones son muy populares, con más de mil millos de pulls en Docker Hub.
+
+#### bullseye
+
+| Criterio              | Ajuste en herramienta | 
+|------------------------|-----------------------|
+| *Mantenimiento* | La última imagen es de hace 5h (visitado el 7 de febrero de 2024). |
+| *Seguridad* | La última versión de esta imagen contiene más de 80 vulnerabilidades, siendo 2 de ellas severas (visitado el 11 de febrero de 2024). |
+
+#### alpine
+
+| Criterio              | Ajuste en herramienta | 
+|------------------------|-----------------------|
+| *Mantenimiento* | La última imagen es de hace 5h (visitado el 7 de febrero de 2024). |
+| *Seguridad* | La última versión de esta imagen no tiene vulnerabilidades conocidas en DockerHub. |
+
+## [Jumpserver](https://hub.docker.com/r/jumpserver/golang)
+
+Esta imagen, del grupo *Jumpserver*, es una imagen mantenida por la comunidad, construida sobre Debian.
+Está patrocinada por Docker a través de su programa de patrocinio a proyectos de Software Libre.
+
+| Criterio              | Ajuste en herramienta | 
+|------------------------|-----------------------|
+| *Popularidad* | La imagen tiene alrededor de 1600 pulls en Docker Hub. |
+| *Mantenimiento* | La última imagen es de hace 3 meses (visitado el 11 de febrero de 2024). |
+| *Seguridad* | No hay ninguna información acerca de la seguridad de esta imagen. |
+
+## Tamaño
+
+Para comprobar el tamaño de las imágenes, se ha ejecutado `docker pull <nombre_imagen>` sobre cada
```

Comment:
Ah, OK que lo has puesto aquí. Habría sido más sencillo en otro lugar, pero en todo caso, ¿lo importante no será el tamaño una vez instalado `just` y el resto de los cambios en la imagen?
Por otro lado, si el tamaño es el principal criterio, ¿has mirado cuantas imágenes oficiales o de vendors oficiales de Alpine y Go hay en Docker Hub y qué tamaño tiene?
Lo que se busca en este objetivo no es que hagáis una imagen específica para un problema específico; se busca que aprendáis cómo se hace este tipo de cosas. Una de la mejor forma de hacerlo es buscar diferentes imágenes, ver como están construidas, y la vayas a elegir o no puedes ver diferentes patrones que te pueden ser interesantes para aplicarlos en tu trabajo.